awx/main/management/commands/inventory_import.py

```diff
@@ -481,12 +481,6 @@ def _delete_groups(self):
             group_names = all_group_names[offset:(offset + self._batch_size)]
             for group_pk in groups_qs.filter(name__in=group_names).values_list('pk', flat=True):
                 del_group_pks.discard(group_pk)
-        if self.inventory_source.deprecated_group_id in del_group_pks:  # TODO: remove in 3.3
-            logger.warning(
-                'Group "%s" from v1 API is not deleted by overwrite',
-                self.inventory_source.deprecated_group.name
-            )
-            del_group_pks.discard(self.inventory_source.deprecated_group_id)
```

Comment:
I think it's a good idea to not delete `deprecated_group` field just yet. We're going to need to talk about the expected behavior of the old models.